modular_zubbers/code/modules/research/techweb/all_nodes.dm

```diff
@@ -66,3 +66,37 @@
 	design_ids -= "mag_autorifle_ic"
 	design_ids += "wt550_ammo_incendiary"
 	. = ..()
+
+/datum/techweb_node/nerd
+	id = "nerd"
+	display_name = "Theoretical Physics"
+	description = "They asked me how well I understood theoretical physics. I said I had a theoretical degree in physics."
+	prereq_ids = list(
```

Comment:
I think basic weapon tech should be added to this aswell

Reply:
"gravity_gun" requires "adv_weaponry" already